packages/xs-extra/xenops-cli.master/opam

```diff
@@ -1,27 +1,28 @@
 opam-version: "2.0"
+name: "xenops-cli"
+version: "master"
+synopsis: "A simple command-line tool for interacting with xenopsd"
+description: "A simple command-line tool for interacting with xenopsd"
 maintainer: "xen-api@lists.xen.org"
-authors: [ "xen-api@lists.xen.org" ]
+authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenops-cli"
 bug-reports: "https://github.com/xapi-project/xenops-cli/issues"
-dev-repo: "git+https://github.com/xapi-project/xenops-cli.git"
-build: [
-  ["dune" "build" "-p" name]
-  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
-]
 depends: [
   "ocaml"
   "dune" {build}
-	"cmdliner"
-	"re"
-	"rpclib"
-	"base-threads"
-	"uuidm"
-	"xapi-idl"
+  "cmdliner"
+  "re"
+  "rpclib"
+  "base-threads"
+  "uuidm"
+  "xapi-idl"
 ]
-synopsis: "A simple command-line tool for interacting with xenopsd"
-description: """
-A simple command-line tool for interacting with xenopsd
-"""
+build: [
+  ["dune" "build" "-p" name]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+dev-repo: "git+https://github.com/xapi-project/xenops-cli.git"
 url {
-  src: "https://github.com/xapi-project/xenops-cli/archive/master/master.tar.gz"
+  src:
+    "https://github.com/xapi-project/xenops-cli/archive/master/master.tar.gz"
```

Comment:
This is out of date, it should point to xenopsd now https://github.com/xapi-project/xenopsd/pull/692

Reply:
https://github.com/xapi-project/xs-opam/blob/master/tools/generate-opam-sources.sh#L68needs to be updated as well to reflect this